app/models/osbb.rb

```diff
@@ -9,6 +9,8 @@ class Osbb < ApplicationRecord
   validates :email,  length: { maximum: 255 },
                      format: { with: VALID_EMAIL_REGEX,
                                message: 'format is not valid' }
+  has_many :members, class_name: :User, dependent: :nullify
+  has_one :lead, class_name: :User, dependent: :nullify
```

Comment:
make it this way `class_name: "User"`

association lead should be user with role LEAD

Reply:
refactor